tests/test_writer.py

```diff
@@ -1675,3 +1675,68 @@ def test_damaged_pdf_length_returning_none():
     reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
     writer = PdfWriter()
     writer.append(reader)
+
+
+@pytest.mark.enable_socket()
+def test_viewerpreferences():
+    """
+    Add Tests for ViewerPreferences
+    https://github.com/py-pdf/pypdf/issues/140#issuecomment-1685380549
+    """
+    url = "https://github.com/py-pdf/pypdf/files/9175966/2015._pb_decode_pg0.pdf"
+    name = "2015._pb_decode_pg0.pdf"
+    reader = PdfReader(BytesIO(get_data_from_url(url, name=name)))
+    v = reader.viewer_preferences
+    assert v.center_window == True  # noqa: E712
```

Comment:
```suggestion
    assert v.center_window is True
```

Reply:
This does not work : the converter forces all the time this but it fails the test

Reply:
Ah, damn. We might have conflicting rules.

Reply:
for log/other users:
This is not a conflicting rules but just the fact that BooleanObjects are different from bool object therefore "is" not working.  ruff is forcing by default the rewriting  so I've added the #noqa. this is more likely a "bug" in ruff to monitor.